test/system/080-pause.bats

```diff
@@ -86,4 +86,42 @@ load helpers
 
     run_podman rm -t 0 -f $cname $cname_notrunning
 }
+
+# bats test_tags=ci:parallel
+@test "podman pause/unpause with HealthCheck interval" {
+    if is_rootless && ! is_cgroupsv2; then
+        skip "'podman pause' (rootless) only works with cgroups v2"
+    fi
+
+    local rootless_flag=""
+    if is_rootless; then
+        rootless_flag="--user"
+    fi
+
+    local ctrname="c-$(safename)"
+    local msg="healthmsg-$(random_string)"
+
+    run_podman run -d --name $ctrname     \
+                --health-cmd "echo $msg"  \
+                --health-interval 1s      \
+                $IMAGE /home/podman/pause
+    cid="$output"
+
+    run systemctl $rootless_flag status $cid-*.{service,timer}
+    assert "$output" =~ "active \(running\)" "service should be running"
+
+    run_podman --noout pause $ctrname
+    assert "$output" == "" "output should be empty"
+
+    run systemctl $rootless_flag status $cid-*.{service,timer}
+    assert "$output" == "" "service should not be running"
+
+    run_podman --noout unpause $ctrname
+    assert "$output" == "" "output should be empty"
+
+    run systemctl $rootless_flag status $cid-*.{service,timer}
+    assert "$output" =~ "active \(running\)" "service should be running"
+
```

Comment:
Maybe let the healthcheck pass once and check to see if it's run successfully here?